cmd/tusd/cli/log.go

```diff
@@ -2,14 +2,14 @@ package cli
 
 import (
 	"log"
-	"os"
 
 	"github.com/tus/tusd"
 )
 
-var stdout = log.New(os.Stdout, "[tusd] ", 0)
-var stderr = log.New(os.Stderr, "[tusd] ", 0)
-
-func logEv(eventName string, details ...string) {
-	tusd.LogEvent(stderr, eventName, details...)
+func logEv(logOutput *log.Logger, eventName string, details ...string) {
+	if logOutput == nil {
```

Comment:
I don't think that this check is needed since you updated all usages of logEv.

Reply:
Yep, definitely unneeded.